local/code/modules/art/paintings.dm

```diff
@@ -1,11 +1,3 @@
-/obj/structure/sign/painting/dorm_private // The REAL exclusive stuff.
-	name = "\improper Private Painting mounting"
-	desc = "For art pieces that fit only in the fine walls of a home. Or a poor excuse for one."
-	desc_with_canvas = "A painting probably taken off the wall with guests over."
-	/// Future maintainability note: AI portrait pickers used to discriminate based on persistence_id but that was commented out. If that's re-implemented this may need checking.
-	/// If you're coming here years in the future and this has been re-implemented before you even found out what SS13 was: lol
-	persistence_id = "dorms_private"
```

Comment:
Signing off on this if we don't plan to migrate our library data *only*.
If we do plan to migrate; it would best to audit & cull it all.

Reply:
i forget that too; it's why i'm conceptually memetic